Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,511 @@
+        secondScreen->addWidget(m_secondDropdown);
+
+        m_thirdDropdown = new FormComboBoxWidget(tr("Optional Global Gem Tag"));
+        AddDropdownActions(m_thirdDropdown);
+        secondScreen->addWidget(m_thirdDropdown);
+
+        m_userDefinedGemTags = new FormLineEditWidget(tr("User-defined Gem Tags (Comma separated list)"), "");
+        secondScreen->addWidget(m_userDefinedGemTags);
+
+        m_gemLocation = new FormFolderBrowseEditWidget(tr("Gem Location"), "");
+        QLineEdit* gemLocationLineEdit = m_gemLocation->lineEdit();
+        const char* gemLocationPlaceholderText = R"(C:\O3DE\Project\Gems\YourGem)";
+        gemLocationLineEdit->setPlaceholderText(tr(gemLocationPlaceholderText));
+        secondScreen->addWidget(m_gemLocation);
+        
+        m_gemIconPath = new FormFolderBrowseEditWidget(tr("Gem Icon Path"), "");
+        QLineEdit* gemIconPathLineEdit = m_gemIconPath->lineEdit();
+        gemIconPathLineEdit->setPlaceholderText(tr("Select Gem icon path"));
+        secondScreen->addWidget(m_gemIconPath);
+
+        m_documentationURL = new FormLineEditWidget(tr("Documentation URL"), "");
+        QLineEdit* documentationURLLineEdit = m_documentationURL->lineEdit();
+        documentationURLLineEdit->setPlaceholderText(tr("Link to any documentation of your Gem i.e. https://o3de.org/docs/user-guide/gems/..."));
+        secondScreen->addWidget(m_documentationURL);
+
+        return secondScreenScrollArea;
+    }
+
+    QScrollArea* CreateAGemScreen::CreateThirdScreen()
+    {
+        QScrollArea* thirdScreenScrollArea = new QScrollArea(this);
+        thirdScreenScrollArea->setWidgetResizable(true);
+        thirdScreenScrollArea->setMinimumWidth(660);
+        thirdScreenScrollArea->setObjectName("createAGemRightPane");
+        QFrame* thirdScreenFrame = new QFrame();
+
+        QWidget* thirdScreenScrollWidget = new QWidget();
+        thirdScreenScrollArea->setWidget(thirdScreenScrollWidget);
+
+        QVBoxLayout* thirdScreen = new QVBoxLayout();
+        thirdScreen->setAlignment(Qt::AlignTop);
+        thirdScreen->addWidget(thirdScreenFrame);
+        thirdScreenScrollWidget->setLayout(thirdScreen);
+
+        QLabel* thirdRightPaneHeader = new QLabel(this);
+        thirdRightPaneHeader->setObjectName("rightPaneHeader");
+        thirdRightPaneHeader->setText(tr("Enter your Details"));
+        thirdScreen->addWidget(thirdRightPaneHeader);
+
+        m_creatorName = new FormLineEditWidget(tr("Creator Name*"), "", this);
+        QLineEdit* creatorNameLineEdit = m_creatorName->lineEdit();
+        creatorNameLineEdit->setPlaceholderText(tr("John Smith"));
+        thirdScreen->addWidget(m_creatorName);
+
+        m_repositoryURL = new FormLineEditWidget(tr("Repository URL*"), "", this);
+        QLineEdit* repositoryURLLineEdit = m_repositoryURL->lineEdit();
+        repositoryURLLineEdit->setPlaceholderText(tr("https://github.com/Jane"));
+        thirdScreen->addWidget(m_repositoryURL);
```

Comment:
This field should not be required and we shouldn't have a placeholder of "Jane"
```suggestion
        m_repositoryURL = new FormLineEditWidget(tr("Repository URL"));
        m_repositoryURL->lineEdit()->setPlaceholderText(tr("Optional URL of the repository for this gem."));
        thirdScreen->addWidget(m_repositoryURL);
```

Reply:
Ahh good catch, looked through this lots of times and never even spotted this default text